Library/Formula/cassandra.rb

```diff
@@ -86,6 +87,38 @@ def install
               %r{`dirname "?\$0"?`/cassandra.in.sh},
               "#{share}/cassandra.in.sh"
 
+    # Prepare tools path
+    mkpath "#{libexec}/tools/bin"
+    rm Dir["#{buildpath}/tools/bin/*.bat"]
+
+    copy Dir["#{buildpath}/tools/bin/*"], "#{libexec}/tools/bin"
```

Comment:
You should be able to use `(libexec/"tools/bin").install Dir[buildpath/"tools/bin/*"]` I'd imagine here.